Angular/src/app/app.module.ts

```diff
@@ -1,5 +1,5 @@
-import { BrowserModule } from '@angular/platform-browser';
-import { NgModule, ErrorHandler } from '@angular/core';
+import { BrowserModule } from "@angular/platform-browser";
+import { NgModule, ErrorHandler } from "@angular/core";
```

Comment:
quotes style should be the same:
```suggestion
import { NgModule, ErrorHandler } from '@angular/core';
```